prio/encrypt.c

```diff
@@ -26,7 +26,10 @@
 #define PRIO_TAG "PrioPacket"
 #define AAD_LEN (strlen(PRIO_TAG) + CURVE25519_KEY_LEN + GCM_IV_LEN_BYTES)
 
-// The all-zeros curve25519 public key, as DER-encoded SKPI blob.
+// For an example import/export code, see:
+// https://searchfox.org/nss/source/gtests/pk11_gtest/pk11_curve25519_unittest.cc#66
```

Comment:
Should be the permalink including the hash, in case the code changes:

https://searchfox.org/nss/rev/cfd5fcba7efbfe116e2c08848075240ec3a92718/gtests/pk11_gtest/pk11_curve25519_unittest.cc#66

Reply:
Good idea. Fixed.